src/material/autocomplete/autocomplete.ts

```diff
@@ -336,4 +343,37 @@ export class MatAutocomplete extends _MatAutocompleteBase {
   @ContentChildren(MatOption, {descendants: true}) options: QueryList<MatOption>;
   protected _visibleClass = 'mat-mdc-autocomplete-visible';
   protected _hiddenClass = 'mat-mdc-autocomplete-hidden';
+
+  constructor(
+    _changeDetectorRef: ChangeDetectorRef,
+    _elementRef: ElementRef<HTMLElement>,
+    @Inject(MAT_AUTOCOMPLETE_DEFAULT_OPTIONS) defaults: MatAutocompleteDefaultOptions,
+    platform?: Platform,
+  ) {
+    super(_changeDetectorRef, _elementRef, defaults, platform);
+
+    this._hideSingleSelectionIndicator = defaults.hideSingleSelectionIndicator ?? false;
+  }
+
+  /** Whether checkmark indicator for single-selection options is hidden. */
+  @Input()
+  get hideSingleSelectionIndicator(): boolean {
+    return this._hideSingleSelectionIndicator;
+  }
+  set hideSingleSelectionIndicator(value: BooleanInput) {
+    this._hideSingleSelectionIndicator = coerceBooleanProperty(value);
+    this._syncParentProperties();
+  }
+  private _hideSingleSelectionIndicator: boolean;
+
+  /** Syncs the parent state with the individual options. */
+  _syncParentProperties(): void {
+    if (this.options) {
+      Promise.resolve().then(() => {
```

Comment:
Why do we need to wrap this in a `Promise.resolve`? I wouldn't have expected it to trigger a "changed after checked" error.

Reply:
Ohh yeah that's right. We don't need the `Promise.resolve` here. I removed it.